nemo/backends/pytorch/nm.py

```diff
@@ -6,7 +6,7 @@
 import torch as t
 import torch.nn as nn
 
-from nemo.core import DeviceType, NeuralModule, WeightShareTransform
+from nemo.core import DeviceType, ModuleType, NeuralModule, WeightShareTransform
```

Comment:
If you introduce this enum, you also need to set this property in NonTrainableNM

Reply:
done